scipy/optimize/_highs/cython/src/_highs_wrapper.pyx

```diff
@@ -1,736 +0,0 @@
-            'time_limit',
-            'mip_rel_gap'
-    ]):
-        val = options.get(opt, None)
-        if val is not None:
-            if not isinstance(val, numbers.Number):
-                warn(_opt_warning(opt.encode(), val), OptimizeWarning)
-            else:
-                opt_status = highs.setHighsOptionValueDbl(opt.encode(), val)
-                if opt_status != HighsStatusOK:
-                    warn(_opt_warning(opt.encode(), val), OptimizeWarning)
-
-
-    # Do all the strings
-    for opt in set(['solver']):
-        val = options.get(opt, None)
-        if val is not None:
-            if not isinstance(val, str):
-                warn(_opt_warning(opt.encode(), val), OptimizeWarning)
-            else:
-                opt_status = highs.setHighsOptionValueStr(opt.encode(), val.encode())
-                if opt_status != HighsStatusOK:
-                    warn(_opt_warning(opt.encode(), val), OptimizeWarning)
-
-
-    # Do all the bool to strings
-    for opt in set([
-            'parallel',
-            'presolve',
-    ]):
-        val = options.get(opt, None)
-        if val is not None:
-            if isinstance(val, bool):
-                if val:
-                    val0 = b'on'
-                else:
-                    val0 = b'off'
-                opt_status = highs.setHighsOptionValueStr(opt.encode(), val0)
-                if opt_status != HighsStatusOK:
-                    warn(_opt_warning(opt.encode(), val, valid_set=[True, False]), OptimizeWarning)
-            else:
-                warn(_opt_warning(opt.encode(), val, valid_set=[True, False]), OptimizeWarning)
-
-
-    # Do the actual bools
-    for opt in set([
-            'less_infeasible_DSE_check',
-            'less_infeasible_DSE_choose_row',
-            'log_to_console',
-            'mps_parser_type_free',
-            'output_flag',
-            'run_as_hsol',
-            'run_crossover',
-            'simplex_initial_condition_check',
-            'use_original_HFactor_logic',
-    ]):
-        val = options.get(opt, None)
-        if val is not None:
-            if val in [True, False]:
-                opt_status = highs.setHighsOptionValueBool(opt.encode(), val)
-                if opt_status != HighsStatusOK:
-                    warn(_opt_warning(opt.encode(), val), OptimizeWarning)
-            else:
-                warn(_opt_warning(opt.encode(), val), OptimizeWarning)
-
-
-ctypedef HighsVarType* HighsVarType_ptr
-
-
-def _highs_wrapper(
-        double[::1] c,
-        int[::1] astart,
-        int[::1] aindex,
-        double[::1] avalue,
-        double[::1] lhs,
-        double[::1] rhs,
-        double[::1] lb,
-        double[::1] ub,
-        np.uint8_t[::1] integrality,
-        dict options):
-    '''Solve linear programs using HiGHS [1]_.
-
-    Assume problems of the form:
-
-        MIN c.T @ x
-        s.t. lhs <= A @ x <= rhs
-             lb <= x <= ub
-
-    Parameters
-    ----------
```

Comment:
Please replicate this docstring in the new highs-wrapper (adapted if/where necessary)

Reply:
I suspect parts of this docstring were out of date to start with as it was written at the start but forgotten about during several updates (as it's not actually rendered anywhere).  If there are interesting bits for the wrapper specifically (such as undocumented HiGHS developer options), those would be the things to pull forward

Reply:
Given that it is not rendered / verified, would it still make sense to try to port it? @h-vetinari

Reply:
> Given that it is not rendered / verified, would it still make sense to try to port it? @h-vetinari

If it's not used and the actual docstring comes from somewhere else, then I agree that it's best to delete the redundant/outdated one.